packages/mobile/src/screens/chat-screen/LinkPreview.tsx

```diff
@@ -0,0 +1,160 @@
+import { useEffect, useState } from 'react'
+
+import type { GestureResponderEvent } from 'react-native'
+import { View, Image } from 'react-native'
+
+import { Text, Link } from 'app/components/core'
+import { audiusSdk } from 'app/services/audius-sdk'
+import { makeStyles } from 'app/styles'
+
+import { REACTION_LONGPRESS_DELAY } from './constants'
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  root: {
+    backgroundColor: palette.white,
+    borderTopLeftRadius: spacing(3),
+    borderTopRightRadius: spacing(3),
+    overflow: 'hidden'
+  },
+  rootIsLinkPreviewOnly: {
+    borderRadius: spacing(3)
+  },
+  thumbnail: {
+    display: 'flex',
```

Comment:
there is by the way a `flexRowCentered` function that you can import. it's used like this: `...flexRowCentered()`

Reply:
Yeah I should use that...